src/config.cpp

```diff
@@ -90,8 +199,9 @@ void parse_env() {
 
 } // namespace
 
+void init_config_from_env_only() { parse_env(); }
```

Comment:
Why are we now parsing the environment config twice?

Reply:
https://github.com/kpet/clvk/pull/679/files#r1527486001